arm-network/2016-09-01/swagger/network.json

```diff
@@ -6469,6 +6612,79 @@
         }
       }
     },
+    "BgpPeerStatus": {
+      "properties": {
+        "localAddress": {
+          "type": "string",
+          "description": "Gets or sets the virtual network gateway's local address"
+        },
+        "neighbor": {
+          "type": "string",
+          "description": "Gets or sets the remote BGP peer"
+        },
+        "asn": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Gets or sets the autonomous systen number of the remote BGP peer"
+        },
+        "state": {
+          "type": "string",
+          "description": "Gets or sets the BGP peer state (unknown, stopped, idle, connecting, connected)"
+        },
+        "connectedDuration": {
+          "type": "string",
+          "description": "Gets or sets how long the peering has been up"
+        },
+        "routesReceived": {
+          "type": "integer",
+          "format":  "int64",
+          "description": "Gets or sets the number of routes learned from this peer"
+        },
+        "messagesSent": {
+          "type": "integer",
+          "format": "int64",
+          "description": "Gets or sets the number of BGP messages sent"
+        },
+        "messagesReceived": {
+          "type": "integer",
+          "format": "in64",
+          "description":  "Gets or sets the number of BGP messages received"
+        }
+      }
+    },
+    "GatewayRoute": {
+      "properties": {
+        "localAddress": {
+          "type": "string",
+          "description": "Gets or sets the gateway's local address"
+        },
+        "network": {
+          "type": "string",
+          "description": "Gets or sets the route's network prefix"
+        },
+        "nextHop": {
+          "type": "string",
+          "description": "Gets or sets the route's next hop"
```

Comment:
I'm not an expert on this topic but maybe users are neither: Is this an IP address or something else? Maybe make the description more verbose